src/compas/topology/orientation.py

```diff
@@ -88,7 +88,7 @@ def _face_adjacency(vertices, faces, nmax=10, radius=10.0):
                         found.add(nbr)
                         break
 
-        adjacency[face] = nbrs
+        adjacency[index] = nbrs
```

Comment:
no it is correct like this. previous version was wrong. not sure how that happened. `face` is a list of vertices. it is not an identifier. `index` is the index of the list of vertices defining the face in the list of faces. so the adjacency dict is in fact returning identifiers as keys as you say...

Reply:
will correct the docs.

Reply:
> no it is correct like this. previous version was wrong. not sure how that happened. `face` is a list of vertices. it is not an identifier. `index` is the index of the list of vertices defining the face in the list of faces. so the adjacency dict is in fact returning identifiers as keys as you say...

now it is the same as `face_adjacency` a few lines below, where it is perhaps a bit clearer because the enumeration says `for i, vertices in enumerate(faces)` instead of `for index, face in enumerate(faces)`